config/jobs/kubernetes/sig-network/sig-network-kind.yaml

```
@@ -100,6 +100,68 @@ presubmits:
       testgrid-tab-name: pr-sig-network-kind, ipvs, dual
       description: Runs tests against a Dual Stack Kubernetes in Docker cluster
       testgrid-alert-email: antonio.ojea.garcia@gmail.com
+  - name: pull-kubernetes-e2e-kind-cloud-provider-loadbalancer
+    cluster: k8s-infra-prow-build
+    optional: true
+    always_run: false
+    skip_report: false
+    run_if_changed: '(provider|cloud-controller-manager|cloud|ipam|endpoint|pkg\/proxy|test\/e2e\/network|test\/e2e\/storage)'
+    decorate: true
+    path_alias: k8s.io/kubernetes
+    extra_refs:
+    - org: kubernetes-sigs
+      repo: cloud-provider-kind
+      base_ref: main
+      path_alias: sigs.k8s.io/cloud-provider-kind
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    decoration_config:
+      timeout: 60m
+      grace_period: 15m
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/krte:v20240111-cf1d81388e-master
+        command:
+        - runner.sh
+        args:
+        - "/bin/bash"
+        - "-c"
+        # compile cloud-provider-kind with kubernetes changes on the PR
+        - set -o errexit;
+          set -o nounset;
+          set -o pipefail;
+          set -o xtrace;
+          cd $GOPATH/src/sigs.k8s.io/cloud-provider-kind;
+          KUBERNETES_ROOT=$GOPATH/src/k8s.io/kubernetes hack/ci/add-kubernetes-to-workspace.sh;
+          make;
+          chmod +x ./bin/cloud-provider-kind && nohup ./bin/cloud-provider-kind > ccm-kind.log 2>&1 &;
+          curl -sSL https://kind.sigs.k8s.io/dl/latest/linux-amd64.tgz | tar xvfz - -C "${PATH%%:*}/" && $GOPATH/src/sigs.k8s.io/cloud-provider-kind/hack/ci/e2e.sh
+        env:
+        # skip serial tests and run with --ginkgo-parallel
+        - name: "PARALLEL"
+          value: "true"
+        - name: FOCUS
+          value: \[sig-network\]|\[Conformance\]
+        # cloud-provider-kind implement loardbalancer in proxy-mode is it does require NodePort
```

```suggestion
        # cloud-provider-kind implements loadbalancer in Proxy mode; it requires NodePort
```

(ISTM like the cloud provider ought to be able to indicate in the LoadBalancerStatus that it can't fulfill the request, and the e2e test ought to require the cloud provider to set that status if it doesn't implement this functionality...)

At any rate, the test currently has:

    e2eskipper.SkipUnlessProviderIs("gce", "gke", "aws")

so this ought to be skipped anyway?

---

we want to remove those skippers from in tree, but now that I think we will remove the tests entirely (not clear when this will happen though)